iiop-openjdk/src/test/java/org/wildfly/iiop/openjdk/IIOPSubsystemTestCase.java

```diff
@@ -140,59 +131,6 @@ public void testParseSubsystemWithBadChild() throws Exception {
     }
 
     @Test
-    public void testParseSubsystemWithBadAttribute() throws Exception {
-        String subsystemXml =
-                "<subsystem xmlns=\"" + Namespace.CURRENT.getUriString() + "\" bad=\"very_bad\">" +
-                "</subsystem>";
-        try {
-            super.parse(subsystemXml);
-            Assert.fail("Should not have parsed bad attribute");
-        } catch (XMLStreamException expected) {
-        }
-    }
-
-    @Test
-    public void testDescribeHandler() throws Exception {
-        // parse the subsystem xml and install into the first controller.
-        String subsystemXml =
-                "<subsystem xmlns=\"" + Namespace.CURRENT.getUriString() + "\">" +
-                "<orb socket-binding=\"iiop\" ssl-socket-binding=\"iiop-ssl\"/>"+
-                "</subsystem>";
-
-        AdditionalInitialization additionalInit = new AdditionalInitialization(){
-            @Override
-            protected void setupController(ControllerInitializer controllerInitializer) {
-                controllerInitializer.addSocketBinding("iiop", 3528);
-                controllerInitializer.addSocketBinding("iiop-ssl", 3529);
-            }
-        };
-
-        KernelServices servicesA = createKernelServicesBuilder(additionalInit)
-                .setSubsystemXml(subsystemXml)
-                .build();
-        // get the model and the describe operations from the first controller.
-        ModelNode modelA = servicesA.readWholeModel();
-        ModelNode describeOp = new ModelNode();
-        describeOp.get(OP).set(DESCRIBE);
-        describeOp.get(OP_ADDR).set(
-                PathAddress.pathAddress(
-                        PathElement.pathElement(SUBSYSTEM, IIOPExtension.SUBSYSTEM_NAME)).toModelNode());
-        List<ModelNode> operations = checkResultAndGetContents(servicesA.executeOperation(describeOp)).asList();
-        servicesA.shutdown();
-
-        Assert.assertEquals(1, operations.size());
-
-        // install the describe options from the first controller into a second controller.
-        KernelServices servicesB = createKernelServicesBuilder(additionalInit).setBootOperations(operations).build();
-        ModelNode modelB = servicesB.readWholeModel();
-        servicesB.shutdown();
-
-        // make sure the models from the two controllers are identical.
-        super.compare(modelA, modelB);
-
-    }
```

Comment:
This is all part of AbstractSubsystemBaseTest, but since we had some old guide on how to write subystems, people just copy & pasted that tests into their new subsystems. 
In short it is removed as it is redundant.